src/Microphysics2M.jl

```diff
@@ -545,12 +544,37 @@ function effective_radius(
     Br =
         (xr == 0) ? 0 :
         ((SF.gamma((νr + 1) / μr) * xr / SF.gamma((νr + 2) / μr))^(-μr))
+    Ac = (N_liq * Bc^(νc + 1)) / SF.gamma(νc + 1)
+    Ar = (μr * N_rai * Br^((νr + 1) / μr)) / SF.gamma((νr + 1) / μr)
+
+    cloud_3moment = (Bc == 0) ? 0 : (6 * Ac * C^3 / (Bc * C)^4)
+    cloud_2moment =
+        (Bc == 0) ? 0 : (Ac * SF.gamma(11 / 3) / (Bc^(11 / 3) * C^(2 / 3)))
+    rain_3moment = (Br == 0) ? 0 : (24 * Ar / (Br^4 * C))
+    rain_2moment = (Br == 0) ? 0 : (6 * Ar / (Br^3 * C^(2 / 3)))
+
+    return ((cloud_2moment + rain_2moment) == 0) ? 0 :
+           ((cloud_3moment + rain_3moment) / (cloud_2moment + rain_2moment)) *
+           1e-3
+end
+
+function effective_radius(
```

Comment:
Maybe it would be better to name this function `effective_radius_Liu_Hallet_97` (or something among those lines. This way no-one will use it by accident, thinking they are using the size distribution one.

Reply:
Then it would also need a docstring. I think just moving the docstring you have in lines 515 is ok